pkgs/top-level/python-packages.nix

```
@@ -486,6 +486,8 @@ in {
 
   asyncio-throttle = callPackage ../development/python-modules/asyncio-throttle { };
 
+  asyncio_mqtt = callPackage ../development/python-modules/asyncio_mqtt { };
```

```suggestion
  asyncio-mqtt = callPackage ../development/python-modules/asyncio_mqtt { };
```

PEPXXX requires that we convert _ to -.

---

Updated.